ceda_status_validator/validate.py

```diff
@@ -0,0 +1,11 @@
+from model import StatusPage
+from pydantic import ValidationError
+
+with open("status.json", "r") as status_file:
+    status_page = status_file.read()
+
+try:
+    StatusPage.model_validate_json(status_page)
+except ValidationError as err:
```

Comment:
Why do you catch the error and then turn it into a exit(1)?

I think if you just let the error happen, python will exit with a non-zero status code?

Reply:
Consider using sys.exit?

Reply:
For reference, sys.exit() is better because the site modules might not be available

https://pylint.pycqa.org/en/v2.17.7/user_guide/messages/refactor/consider-using-sys-exit.html